doc/developer-guide/api/functions/TSLifecycleHookAdd.en.rst

```diff
@@ -98,6 +98,13 @@ Types
 
       Called after every SSL context initialization used by |TS| for inbound connections (|TS| as the server).
 
+   .. cpp:enumerator:: TS_LIFECYCLE_TASK_THREADS_READY_HOOK
+
+      Called after |TS| task threads have been started.
+
+      Invoked with the event :c:data:`TS_EVENT_LIFECYCLE_TASK_THREADS_READY` and ``NULL``
+      data.
+
```

Comment:
Are the lifecycle hooks always synchronous?  If not, should put a comment that this plugin hook cannot be delayed.